src/Component/Kernel/src/DependencyInjection/AddCacheClearerPass.js

```diff
@@ -0,0 +1,41 @@
+const CompilerPassInterface = Jymfony.Component.DependencyInjection.Compiler.CompilerPassInterface;
+const Reference = Jymfony.Component.DependencyInjection.Reference;
+
+/**
+ * @memberOf Jymfony.Component.DependencyInjection.Compiler
+ */
+class AddCacheClearerPass extends implementationOf(CompilerPassInterface) {
+    __construct(cacheClearerId = 'cache_clearer', cacheClearerTag = 'kernel.cache_clearer') {
```

Comment:
maybe we need to implement `ChainCacheClearer` and define cache_clearer into service.js